scripts/core/exp-command-buffer.yml

```diff
@@ -0,0 +1,246 @@
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Command Buffer Extension APIs"
```

Comment:
Nitpick, but preference for consistently hyphenating "command buffer" as command-buffer